editions/tw5.com/tiddlers/concepts/WikiText.tid

```diff
@@ -1,15 +1,23 @@
 created: 20131205155227468
-modified: 20140919191220377
+list: [[Headings in WikiText]] [[Paragraphs in WikiText]] [[Hard Linebreaks in WikiText]] [[Formatting in WikiText]] [[Dashes in WikiText]] [[Block Quotes in WikiText]] [[Lists in WikiText]] [[Definitions in WikiText]] [[Tables in WikiText]] [[Code Blocks in WikiText]] [[Typed Blocks in WikiText]] [[Images in WikiText]] [[Horizontal Rules in WikiText]] [[Linking in WikiText]] [[Variables in WikiText]] [[Macro Definitions in WikiText]] [[Macro Calls in WikiText]] [[Macros in WikiText]] [[Transclusion in WikiText]] [[Transclusion and Substitution]] [[Styles and Classes in WikiText]] [[HTML in WikiText]] [[Widgets in WikiText]] [[WikiText Parser Modes]]
+modified: 20220726132727329
 tags: Concepts Reference
 title: WikiText
 type: text/vnd.tiddlywiki
 
+\define openAll()
```

Comment:
If this is a useful feature then it should be made universally available, and not just buried in one particular docs tiddler.

One of our key goals with the documentation is consistency. The value of consistency across the docs more than outweighs the advantages of a random, isolated improvement like this.